apps/fluent-tester/src/RNTester/TestComponents/index.ts

```diff
@@ -12,21 +12,19 @@ import { SeparatorTest } from './Separator';
 import { SvgTest } from './Svg';
 import { TextTest } from './Text';
 import { ThemeTest } from './Theme';
-import {
-  HOMEPAGE_BUTTON_BUTTON,
-  HOMEPAGE_CALLOUT_BUTTON,
-  HOMEPAGE_CHECKBOX_BUTTON,
-  HOMEPAGE_FOCUSTRAPZONE_BUTTON,
-  HOMEPAGE_LINK_BUTTON,
-  HOMEPAGE_PERSONA_BUTTON,
-  HOMEPAGE_PERSONACOIN_BUTTON,
-  HOMEPAGE_PRESSABLE_BUTTON,
-  HOMEPAGE_RADIOGROUP_BUTTON,
-  HOMEPAGE_SEPARATOR_BUTTON,
-  HOMEPAGE_SVG_BUTTON,
-  HOMEPAGE_TEXT_BUTTON,
-  HOMEPAGE_THEME_BUTTON
-} from '../Consts';
+import { HOMEPAGE_CHECKBOX_BUTTON } from './Checkbox/consts';
```

Comment:
could re-export from ./Checkbox, save some lines ?

Reply:
Changed in newest iteration, good change :)